lib/cog/command/pipeline/executor.ex

```diff
@@ -153,12 +158,11 @@ defmodule Cog.Command.Pipeline.Executor do
   to `remaining` in the state.
   """
   def parse(:timeout, state) do
-    case Parser.scan_and_parse(state.request["text"], command_resolver: &BundleResolver.find_bundle/1) do
-      {:ok, %Ast.Invocation{}=invocation} ->
-        {:next_state, :lookup_redirects, %{state | pipeline: %Ast.Pipeline{invocations: [invocation]}}, 0}
+    case Parser.scan_and_parse(state.request["text"], command_resolver: &CommandResolver.find_bundle/1, return_pipeline: true) do
       {:ok, %Ast.Pipeline{}=pipeline} ->
+        maybe_warn_on_ambiguous_aliases(pipeline, state)
```

Comment:
@kevsmith @christophermaier Any thoughts on this? If a command is executed with an ambiguous alias we send a warning to the user. It's a bit odd because we query for aliases on every command invocation, and it won't trigger if the user passes the fully qualified name for the alias. implementation is on L#809 and in the `Lib.Command.AmbiguousAliasChecker` module.